app/src/main/java/com/github/se/orator/ui/offline/RecordingReviewScreen.kt

```diff
@@ -56,7 +56,7 @@ fun RecordingReviewScreen(
             modifier = Modifier.fillMaxWidth().testTag("Back"),
             verticalAlignment = Alignment.CenterVertically) {
               Icon(
-                  imageVector = Icons.Filled.ArrowBack,
+                  imageVector = Icons.Outlined.ArrowBackIosNew,
```

Comment:
I noticed that you changed the arrow back icon on an offline related screen, which seems unrelated to the purpose of this PR. This is a good change as it improves consistency or clarity in the UI. However, while it’s a minor change, it might be better to keep the focus of the PR strictly on its intended scope to maintain clarity in the review process.

Reply:
The code didn't compile due to inconsistencies with the icons after pulling from the main, so I had to change this in order to resolve the issue.